datapreview.tex

```diff
@@ -0,0 +1,126 @@
+
+\subsubsection{Data Preview 0.1}
+
+Data Preview 0.1 (DP0.1) was released to a group, approximately 300,  of early adopters from the community in June 2021. 
+It is based on the the simulated, LSST-like images generated by the Dark Energy Science Collaboration (DESC) for their Data Challenge 2 (DC2), \citep{2021ApJS..253...31L}. 
+DP0.1 only uses the 300~\sqdeg of DC2 images that were simulated for five years of the LSST’s wide-fast-deep component (WFD) using a baseline cadence, \citedsp{PSTN-055}.
+The DESC processed the simulated DC2 images with \href{https://pipelines.lsst.io/v/v19_0_0/index.html}{Version 19} of the LSST Science Pipelines. 
+DP0.1 makes the DESC’s DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 
+
+For full details on DP0.1 including an exact description of the data products served, see the documentation at \url{https://dp0-1.lsst.io/}
+
+\subsubsection{Data Preview 0.2}
+
+Data Preview 0.2 (DP0.2) was released to approximately 600 early adopters from the community in June 2022, exactly 1 year after DP0.1. 
+The dataset used for DP0.2 was the same as that used for DP0.1.
+Rubin processed the simulated DC2 images with \href{https://pipelines.lsst.io/v/v23_0_0/index.html}{Version 23} of the LSST Science Pipelines. 
+DP0.2 makes the Rubin reprocessed DESC DC2 images and catalogs available to users through an early version the Rubin Science Platform (RSP) running at the US DAC. 
+
+For full details on DP0.2 including an exact description of the data products served, see the documentation at \url{https://dp0-2.lsst.io/}
+
+\subsubsection{Data Preview 0.3}
+
+Scheduled for between June and September 2023, DP0.3 will be the last in the DP0 series of Data Previews based on simulated LSST-like data. 
+The main goal of DP0.3 is to support the Solar System Science Collaboration by hosting their simulated 10-year catalog and serving it via the RSP at the US DAC. 
+Table~\ref{tab:dp-zero-three-products} presents a summary of the expected DP0.3 data products, as of January 2023.
+The final exact data products for DP0.3 are still to be decided. 
+
+\subsection{Data Preview 1}
+
+Data Preview 1 was originally defined to be based on reprocessed on-sky data taken with ComCam.
+Following the replan of the Construction project in December 2022, no on-sky data will be taken with ComCam, \S~\ref{sec:commissioning}.
+Consequently DP1 has been redefined to be based on a subset of science-grade images taken with LSSTCam during a period of a few days around the System First Light milestone, \S~\ref{ssec:commissioning-observations}.
+The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
+As a minimal set of data products, visit-level images and catalogs to enable initial studies of observational and instrumental effects will be released. 
```

Comment:
It's not clear to me if "As a minimal set of data products," means that these are the minimum and we might opt to produce more if possible, or if it's describing these visit-level images and catalogs as being a somewhat limited set (compared to the full DR products)

Reply:
I mean the former. I will try to rephrase to be clearer - suggestions welcome

Reply:
How about: 
"The processing pipelines and exact data products that will comprise DP1 are still under discussion. 
At minimum, DP1 will deliver visit-level images and catalogs to enable initial studies of observational and instrumental effects will be released. 
"

Reply:
That would be clear to me.

Reply:
Thanks!